analyses/MI_cd_2010/doc_MI_cd_2010.md

```diff
@@ -0,0 +1,24 @@
+# 2010 Michigan Congressional Districts
+
+## Redistricting requirements
+in Michigan, according to (http://www.legislature.mi.gov/(S(xxvumgge0jwzkeswmwt0bh4v))/mileg.aspx?page=GetObject&objectname=mcl-Article-IV-6), districts must:
+
+1. be contiguous
+2. have equal populations
+3. be geographically compact
+4. preserve county and municipality boundaries as much as possible
+5. cannot favor/disfavor incumbents
+
+
+### Algorithmic Constraints
+We enforce a maximum population deviation of 0.5%. We applied a constraint to limit county and municipality splits (see '02_setup_MI_cd_2010.R' file).
```

Comment:
Change 2nd sentence to `We use a pseudo-county constraint to help preserve county and municipality boundaries, as described below.`